app/src/main/java/com/github/se/orator/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -20,10 +24,13 @@ fun BottomNavigationMenu(
     tabList: List<TopLevelDestination>,
     selectedItem: String
 ) {
+  val insets = WindowInsets.systemBars.asPaddingValues()
+
   BottomNavigation(
       modifier =
           Modifier.fillMaxWidth()
-              .height(AppDimensions.bottomNavigationHeight)
+              .height(AppDimensions.bottomNavigationHeight + insets.calculateBottomPadding())
```

Comment:
Good job adding the insets.calculateBottomPadding() to dynamically adjust the height! This ensures proper spacing and alignment with the system navigation bar, making the layout more adaptable to different devices and configurations.